src/usage_metrics/models.py

```diff
@@ -466,6 +466,144 @@
     Column("partition_key", String),
 )
 
+out_zenodo_logs = Table(
+    "out_zenodo_logs",
```

Comment:
I think it would be helpful to add table descriptions as doc strings or via `Table.comment` so we can easily understand the differences between the core and out zenodo tables.